src/commands/instances/moderation/ReviewNameChange.ts

```diff
@@ -39,7 +32,7 @@ class ReviewNameChange implements Command {
         .setTitle(`Name change review: ${reviewData.oldName} → ${reviewData.newName}`)
         .setDescription(this.buildReviewMessage(reviewData));
 
-      message.respond(response);
+      message.reply({ embeds: [response] });
```

Comment:
`await` this: This way if there is an `error` it will be caught in the try/catch, instead of a rogue failed promise.